pages/crash_stats_per_day.py

```diff
@@ -11,7 +11,7 @@
 from pages.base_page import CrashStatsBasePage
 
 
-class CrashStatsPerActiveDailyUser(CrashStatsBasePage):
+class CrashStatsPerDay(CrashStatsBasePage):
 
     _page_title = 'Crashes per Active Daily User for Firefox'
```

Comment:
Is this title still accurate now that the report name has changed? If so, then maybe that's a bug that needs to be filed? I would have expected the title to change along with the report name.